docs/reference/eql/eql.asciidoc

```diff
@@ -50,9 +50,18 @@ query>>. If the {es} {security-features} are enabled, you must have the `read`
 <<privileges-list-indices,index privilege>> for the target data stream, index,
 or index alias.
 
+////
 [source,console]
 ----
-GET /my-index-000001/_eql/search
+DELETE /_data_stream/*
+DELETE /_index_template/*
+----
+// TEARDOWN
+////
```

Comment:
I'll replace this with a reusable teardown once https://github.com/elastic/elasticsearch/pull/70831 is merged.